File: Beginner/1061 - Event Time/1061.c
#include <stdio.h>

int converte(int day, int hour, int min, int sec) {
    return (24 * 60 * 60 * day) + (60 * 60 * hour) + (60 * min) + sec;
}

int main() {
    int d1, h1, m1, s1, d2, h2, m2, s2;
    scanf("Dia %d\n", &d1);
    scanf("%d : %d : %d\n", &h1, &m1, &s1);
    scanf("Dia %d\n", &d2);
    scanf("%d : %d : %d\n", &h2, &m2, &s2);

    int secTotal = converte(d2, h2, m2, s2) - converte(d1, h1, m1, s1);

    printf("%d dia(s)\n", secTotal / (24 * 60 * 60));
    secTotal %= 24 * 60 * 60;
    printf("%d hora(s)\n", secTotal / (60 * 60));
    secTotal %= 60 * 60;
    printf("%d minuto(s)\n", secTotal / 60);
    secTotal %= 60;
    printf("%d segundo(s)\n", secTotal);

    return 0;
}

File: Beginner/1061 - Event Time/1061.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string[] entrada;
        int d1, h1, m1, s1, d2, h2, m2, s2;

        d1 = int.Parse(Console.ReadLine().Trim().Split(' ')[1]);
        entrada = Console.ReadLine().Trim().Split(':');
        h1 = int.Parse(entrada[0].Trim());
        m1 = int.Parse(entrada[1].Trim());
        s1 = int.Parse(entrada[2].Trim());
        d2 = int.Parse(Console.ReadLine().Trim().Split(' ')[1]);
        entrada = Console.ReadLine().Trim().Split(':');
        h2 = int.Parse(entrada[0].Trim());
        m2 = int.Parse(entrada[1].Trim());
        s2 = int.Parse(entrada[2].Trim());

        int secTotal = ((24 * 60 * 60 * d2) + (60 * 60 * h2) + (60 * m2) + s2) - ((24 * 60 * 60 * d1) + (60 * 60 * h1) + (60 * m1) + s1);

        Console.WriteLine($"{secTotal / (24 * 60 * 60)} dia(s)");
        secTotal %= 24 * 60 * 60;
        Console.WriteLine($"{secTotal / (60 * 60)} hora(s)");
        secTotal %= 60 * 60;
        Console.WriteLine($"{secTotal / 60} minuto(s)");
        secTotal %= 60;
        Console.WriteLine($"{secTotal} segundo(s)");
    }
}